lightning/src/ln/channelmanager.rs

```diff
@@ -6075,6 +6087,8 @@ impl_writeable_tlv_based!(ChannelDetails, {
 	(28, is_funding_locked, required),
 	(30, is_usable, required),
 	(32, is_public, required),
+	(34, htlc_minimum_msat, required),
```

Comment:
We shouldn't be adding new required even values for these, I think, no? Even means "old versions will fail to read" (instead of ignore), and required means we'll fail to read if its missing (ie old versions).

Reply:
Ok thanks, I'll make this an odd value instead to make old versions ignore the field.